src/index.css

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-html, body {
+ html, body {
```

Comment:
why the added trailing space here?

Reply:
Fixed: Copied prettier config from sdk-js and applied it to entire project